### PR TITLE
Update hyprland

### PR DESCRIPTION
**Pull request by gNick**
**Summary**

Update for hyprland packages and add few new packages (hyprland-qutils and hyprgraphics).

**Test Plan**

Tested on my laptop and a VM

**Checklist**

- [x] Package was built and tested against unstable
- [x] This change could gainfully be listed in the weekly sync notes once merged

### CODE REVIEW

File: packages/h/hyprgraphics/package.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name       : hyprgraphics
+version    : 0.1.1
+release    : 1
+source     :
+    - https://github.com/hyprwm/hyprgraphics/archive/refs/tags/v0.1.1.tar.gz : 123a29e53a00bfaf37e7971b929433fb716fd497b3555f72c2d0419ae18e65c7
+homepage   : https://github.com/hyprwm/hyprgraphics
+license    : BSD-3-Clause
+component  : desktop-hyprland
```
Comment (EbonJaeger) [inline packages/h/hyprgraphics/package.yml:8]: Should be `desktop.hyprland`

File: packages/h/hyprgraphics/package.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+summary    : small C++ library with graphics / resource
```
Comment (EbonJaeger) [inline packages/h/hyprgraphics/package.yml:9]: Could we use this for the summary, instead?

```text
Hyprland graphics / resource utilities
```

File: packages/h/hyprland-qutils/package.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name       : hyprland-qutils
+version    : 0.1.1
+release    : 1
+source     :
+    - https://github.com/hyprwm/hyprland-qtutils/archive/refs/tags/v0.1.1.tar.gz : 3d88e7aaea9ad1c5dc7703d43e4021f06e3f582a656a7592708d8204a272236c
+homepage   : https://github.com/hyprwm/hyprland-qtutils
+license    : BSD-3-Clause
+component  : desktop-hyprland
```
Comment (EbonJaeger) [inline packages/h/hyprland-qutils/package.yml:8]: Should be `desktop.hyprland`

File: packages/h/hyprland-qutils/package.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+summary    : qt/qml utilities that might be used by various hypr* apps
```
Comment (EbonJaeger) [inline packages/h/hyprland-qutils/package.yml:9]: Could we use this instead?

```text
Hyprland QT/qml utility apps
```

File: packages/h/hyprland-qutils/package.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+description: |
+    qt/qml utilities that might be used by various hypr* apps
+builddeps  :
+  - pkgconfig(hyprutils)
+  - pkgconfig(Qt6Quick)
+  - pkgconfig(Qt6WaylandClient)
```
Comment (EbonJaeger) [inline packages/h/hyprland-qutils/package.yml:15]: Names starting in lowercase should be below those starting in Uppercase.

File: packages/h/hyprpaper/package.yml [hunk 6]
```diff
@@ -11,16 +11,18 @@ description: |
...
     - pkgconfig(cairo)
+    - pkgconfig(hyprgraphics)
     - pkgconfig(hyprlang)
     - pkgconfig(hyprutils)
     - pkgconfig(hyprwayland-scanner)
     - pkgconfig(libglvnd)
+    - pkgconfig(libjxl)
     - pkgconfig(libmagic)
     - pkgconfig(libturbojpeg)
     - pkgconfig(libwebp)
```
Comment (EbonJaeger) [inline packages/h/hyprpaper/package.yml:22]: It looks like it no longer links against these. Are they still needed?

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Overall looks good, I just have some minor things. :)

File: packages/h/hyprpaper/package.yml (hunk 6 above)
Comment (gNick) [reply packages/h/hyprpaper/package.yml:22]: In the github page of the project these dependecies are still listed ([source](https://github.com/hyprwm/hyprpaper#dependencies)).
I checked in the Arch wiki and they still use these depencies too ([source](https://archlinux.org/packages/extra/x86_64/hyprpaper/)).

I tested to build without libwebp and cmake shows an error message which says libwebp is missing.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**